### PR TITLE
Revert space-saving operations

### PR DESCRIPTION
**Pull request by workingjubilee**
The "all of our artifacts" `mv` seems like it may save enough space to matter sometimes, since it can range up to a gigabyte of difference, if memory serves. For the rest, I think we're good.

r? @Kobzol

try-job: dist-aarch64-apple

### CODE REVIEW

File: src/ci/scripts/upload-artifacts.sh [hunk 1]
```diff
@@ -19,18 +19,18 @@ fi
 if [[ "${DEPLOY-0}" -eq "1" ]] || [[ "${DEPLOY_ALT-0}" -eq "1" ]]; then
     dist_dir="${build_dir}/dist"
     rm -rf "${dist_dir}/doc"
-    mv "${dist_dir}"/* "${upload_dir}"
+    mv -r "${dist_dir}"/* "${upload_dir}"
```
Comment (Kobzol) [inline src/ci/scripts/upload-artifacts.sh:22]: What is `mv -r`? :thinking:

Comment (workingjubilee) [reply src/ci/scripts/upload-artifacts.sh:22]: I somehow have made this mistake twice.

**Pull request merged**